### ISSUE TITLE
🐛 Bug: Accessibility issue on mochajs.org: missing alt text

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [mochajs/mocha#5066]
### Bug Report Checklist

- [X] I have read and agree to Mocha's [Code of Conduct](https://github.com/mochajs/mocha/blob/master/.github/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/mochajs/mocha/blob/master/.github/CONTRIBUTING.md)
- [X] I have searched for [related issues](https://github.com/mochajs/mocha/issues?q=is%3Aissue) and [issues with the `faq` label](https://github.com/mochajs/mocha/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Afaq%20), but none matched my issue.
- [X] I have 'smoke tested' the code to be tested by running it outside the real test suite to get a better sense of whether the problem is in the code under test, my usage of Mocha, or Mocha itself.
- [X] I want to provide a PR to resolve this

### Expected

The website should not have any accessibility issues - including ones detectable by the [axe browser devtools](https://www.deque.com/axe/browser-extensions).

### Actual

The Netlify logo in the footer is [missing image alt text](https://dequeuniversity.com/rules/axe/4.8/image-alt).

https://github.com/mochajs/mocha/blob/509938d89e8f269030ed1d514062cb554efd2a51/docs/_includes/default.liquid#L131-L135

### Minimal, Reproducible Example

1. Visit https://mochajs.org
2. Wait for the page to load & fade in
3. Run the [axe browser devtools](https://www.deque.com/axe/browser-extensions)

### Versions

n/a

### Additional Info

https://web.dev/explore/accessible explains more around web accessibility.

I first ran a [Lighthouse accessibility audits](https://developer.chrome.com/docs/lighthouse/accessibility/scoring) but it didn't wait for the page to fade in and reported many false negatives as a result.
